<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>361</width>
    <height>343</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <widget class="QGroupBox" name="selectBox">
     <property name="title">
      <string>Select Serial Port</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QComboBox" name="serialPortInfoListBox"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="descriptionLabel">
        <property name="text">
         <string>Description:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="manufacturerLabel">
        <property name="text">
         <string>Manufacturer:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="serialNumberLabel">
        <property name="text">
         <string>Serial number:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="locationLabel">
        <property name="text">
         <string>Location:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="vidLabel">
        <property name="text">
         <string>Vendor ID:</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="pidLabel">
        <property name="text">
         <string>Product ID:</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="brgModeLabel">
       <property name="text">
        <string>Interface:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="brgModeBox"/>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>96</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QGroupBox" name="additionalOptionsGroupBox">
     <property name="title">
      <string>Additional options</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QCheckBox" name="logFileCheckBox">
        <property name="text">
         <string>Log to file</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QTabWidget" name="parametersBox">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="parametersBoxPage1">
      <attribute name="title">
       <string>CAN</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="baudRateLabel">
         <property name="text">
          <string>Baud Rate:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="baudRateBox"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="canModeLabel">
         <property name="text">
          <string>CAN Mode:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="canModeBox"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="msg0IdLabel">
         <property name="text">
          <string>MSG(0) CAN_STD_ID:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="msg0StdIdEdit">
         <property name="placeholderText">
          <string>487</string>
         </property>
         <property name="clearButtonEnabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="msg1IdLabel">
         <property name="text">
          <string>MSG(1) CAN_STD_ID:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="msg1StdIdEdit">
         <property name="placeholderText">
          <string>784</string>
         </property>
         <property name="clearButtonEnabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="msg2IdLabel">
         <property name="text">
          <string>MSG(2) CAN_EXT_ID:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="msg2ExtIdEdit">
         <property name="clearButtonEnabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="msg3IdLabel">
         <property name="text">
          <string>MSG(3) CAN_EXT_ID:</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QLineEdit" name="msg3ExtIdEdit">
         <property name="clearButtonEnabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabI2C">
      <attribute name="title">
       <string>I2C</string>
      </attribute>
      <layout class="QGridLayout" name="i2cgridLayout">
       <item row="0" column="1">
        <widget class="QComboBox" name="i2cModeBox"/>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="i2cClockBox"/>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="i2cModeLabel">
         <property name="text">
          <string>I2C Mode:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="i2cClockLabel">
         <property name="text">
          <string>Bus Clock:</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabSPI">
      <attribute name="title">
       <string>SPI</string>
      </attribute>
      <layout class="QGridLayout" name="spigridLayout">
       <item row="0" column="1">
        <widget class="QComboBox" name="spiModeBox"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="spiClockLabel">
         <property name="text">
          <string>Bus Clock:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="spiClockBox"/>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="spiModeLabel">
         <property name="text">
          <string>SPI Mode:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="spiTypeLabel">
         <property name="text">
          <string>Type:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QComboBox" name="spiTypeBox"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="spiOrderLabel">
         <property name="text">
          <string>Order:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QComboBox" name="spiOrderBox"/>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="spiSsActiveLabel">
         <property name="text">
          <string>SS Active:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QComboBox" name="spiSsActiveBox"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabUART">
      <attribute name="title">
       <string>UART</string>
      </attribute>
      <widget class="QLabel" name="uartBaudRateLabel">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>108</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Baud Rate:</string>
       </property>
      </widget>
      <widget class="QComboBox" name="uartBaudRateBox">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>20</y>
         <width>81</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="uartParityBitLabel">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>108</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Parity Bit:</string>
       </property>
      </widget>
      <widget class="QLabel" name="uartDataBitLabel">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>100</y>
         <width>108</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Data Bit:</string>
       </property>
      </widget>
      <widget class="QLabel" name="uartBaudRateLabel_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>140</y>
         <width>108</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Stop Bit:</string>
       </property>
      </widget>
      <widget class="QComboBox" name="uartParityBitBox">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>60</y>
         <width>81</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QComboBox" name="uartDataBitBox">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>100</y>
         <width>81</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QComboBox" name="uartStopBitBox">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>140</y>
         <width>81</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
